src/ansys/aedt/core/modeler/schematic.py

```diff
@@ -23,23 +23,17 @@
 # SOFTWARE.
 
 import random
-import sys
 import time
-import warnings
 
 from ansys.aedt.core.generic.constants import AEDT_UNITS
 from ansys.aedt.core.generic.general_methods import is_linux
 from ansys.aedt.core.generic.general_methods import pyaedt_function_handler
 from ansys.aedt.core.generic.settings import settings
 from ansys.aedt.core.modeler.cad.modeler import Modeler
-
-if (3, 8) < sys.version_info < (3, 12):
-    from ansys.aedt.core.modeler.circuits.primitives_emit import EmitComponent
-    from ansys.aedt.core.modeler.circuits.primitives_emit import EmitComponents
-else:  # pragma: no cover
-    warnings.warn("Emit API is only available for Python 3.8+,<3.12.")
```

Comment:
Unless there is a version compatible with python3.13 (python version that should be available soon), I would like to keep the previous code and change (3.12) into (3.13) and "<3.12" into "3.13".

Reply:
Makes sense - I've made the change and corrected the warning message that was inaccurate before. Thank you.